tmt/steps/report/reportportal.py

```diff
@@ -14,85 +14,124 @@
-        help="Pass the description for ReportPortal launch, especially with '--suite-per-plan' "
-             "option (Otherwise Summary from plan fmf data per each launch is used by default).")
+        metavar="DESCRIPTION",
+        default=_str_env_to_default('launch_description', None),
+        help="Pass the description for ReportPortal launch with '--suite-per-plan' option"
+             " or append the original (plan summary) with additional info."
+             "Appends suite/test description with upload-to-launch/suite options.")
+
     launch_per_plan: bool = field(
         option="--launch-per-plan",
-        default=False,
+        default=_flag_env_to_default('launch_per_plan', False),
         is_flag=True,
         help="Mapping launch per plan, creating one or more launches with no suite structure.")
+
     suite_per_plan: bool = field(
         option="--suite-per-plan",
-        default=False,
+        default=_flag_env_to_default('suite_per_plan', False),
         is_flag=True,
         help="Mapping suite per plan, creating one launch and continous uploading suites into it. "
              "Recommended to use with '--launch' and '--launch-description' options."
              "Can be used with '--upload-to-launch' option to avoid creating a new launch.")
+
     upload_to_launch: Optional[str] = field(
         option="--upload-to-launch",
         metavar="LAUNCH_ID",
-        default=None,
+        default=_str_env_to_default('upload_to_launch', None),
         help="Pass the launch ID for an additional test/suite upload to an existing launch. "
              "ID can be found in the launch URL.")
+
     upload_to_suite: Optional[str] = field(
         option="--upload-to-suite",
         metavar="LAUNCH_SUITE",
-        default=None,
+        default=_str_env_to_default('upload_to_suite', None),
         help="Pass the suite ID for an additional test upload to a suite "
              "within an existing launch. ID can be found in the suite URL.")
+
     launch_rerun: bool = field(
         option="--launch-rerun",
-        default=False,
+        default=_flag_env_to_default('launch_rerun', False),
         is_flag=True,
         help="Rerun the last launch based on its name and unique test paths to create Retry item"
              "with a new version per each test. Supported in 'suite-per-plan' structure only.")
+
     defect_type: Optional[str] = field(
         option="--defect-type",
         metavar="DEFECT_NAME",
-        default=None,
+        default=_str_env_to_default('defect_type', None),
         help="Pass the defect type to be used for failed test, which is defined in the project"
              " (e.g. 'Idle'). 'To Investigate' is used by default.")
+
     exclude_variables: str = field(
         option="--exclude-variables",
         metavar="PATTERN",
-        default="^TMT_.*",
+        default=str(_str_env_to_default('defect_type', "^TMT_.*")),
```

Comment:
`str()` call is superfluous.

Reply:
Aha, it's probably not, it turns `None` into `"None"`. Can the type of `exclude_variables` be `Optional[str]`?

Reply:
Nevermind, that would be wrong as there's a defined default value...

Reply:
See https://github.com/teemtee/tmt/pull/2702#discussion_r1503818742

Reply:
It should never be None in this case. So can I leave it like this, or better change it to Optional[str]?